src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -151,15 +156,16 @@ public void shouldChargeProcessingAndItemFeesWhenBothDefined() {
 
     useLostItemPolicy(lostItemFeePoliciesFixture.create(lostItemPolicy).getId());
 
-    final UUID loanId = createLoanAndDeclareItLost();
+    final IndividualResource loan = createLoanAndDeclareItLost();
```

Comment:
The `beforeEach` method appears to check an item out to a patron before each test starts. Why does another loan need to be created during the test?

Reply:
Fee/fine tests have to create a lost item policy which is specific per a test. In order to apply it we need to create loan each time.

Reply:
What happens to the loan created in the `beforeEach` section? Is it redundant?

Reply:
It is used in few tests, I can modify them to explicitly create loan.